cvmfs/swissknife_graft.cc

```diff
@@ -0,0 +1,249 @@
+/**
+ * This file is part of the CernVM File System.
+ *
+ * Process a set of input files and create appropriate graft files.
+ */
+
+#include <fcntl.h>
+#include <unistd.h>
+#include <sys/stat.h>
+
+#include <cstdio>
+#include <iostream>
```

Comment:
Please avoid `iostream` and `sstream`.  I think it can be replace throughout the code by string concatenation and the `Stringify...` functions from util.cc